x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/defense_evasion_suspicious_certutil_commands.json

```diff
@@ -52,5 +52,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 12
+  "version": 14
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950